package.json

```diff
@@ -89,8 +89,8 @@
     "ember-auto-import": "^2.6.3",
     "ember-cli-babel": "^8.2.0",
     "ember-cli-htmlbars": "^6.3.0",
+    "ember-composable-helpers": "^5.0.0",
```

Comment:
It seems we were using this in the data table template overrides, but we weren't depending on the addon itself. This only worked because ember-data-table brought it in.